Upgrade gradle repositories

oguzkocer opened the pull request:
Fixes #

To test:

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

oguzkocer committed:
Remove repositories section from allprojects

oguzkocer committed:
Update stories-android which updates its gradle repositories

oguzkocer committed:
Update gradle repositories for each module

oguzkocer committed:
Update Buildkite lint task log level to info

AliSoftware commented:
@oguzkocer Is this PR still relevant, or has it been superseeded by another since? Just checking because it's from Aug 2 and I'm pushing its milestone multiple times, so just want to be sure the Draft PR is still on your radar 🙂

oguzkocer commented:
@AliSoftware I am so sorry, I didn't realize I added a milestone to this PR. It's still somewhat relevant, but it's a low impact change and because `jcenter` is still used, I am not liking the complication the PR brings. I've removed the milestone now. If the PR being open won't bother you (or anyone else), I'd rather keep it that way because it'll stay on my radar. However, we can also close the PR if someone is bothered and I'll move it to my todo list.

You also had to move milestones for some other PRs, I am sorry that I wasn't on top of those. I'll try to add the milestone when I am somewhat sure of the state of the PR from now on.

AliSoftware commented:
No worries @oguzkocer! It doesn't bother me 🙂 I just wanted to check you didn't forget about it is all 🙂

Keeping the PR open and removing the milestone makes sense to me 👍
